tests/configs/basic_app/k8s_jobs.yml

```
@@ -24,4 +24,7 @@ spec:
         image: generic-image
         imagePullPolicy: Always
         name: upgrade
+        resources: {
+          requests: { 'cpu': 1, 'memory': '1Gi'}
+        }
```

```suggestion
        resources:
          requests:
            cpu: 1
            memory: "1Gi"
```
I believe this is the correct formatting